var/spack/repos/builtin/packages/epics-base/package.py

```diff
@@ -23,6 +23,10 @@ class EpicsBase(MakefilePackage):
     depends_on("readline")
     depends_on("perl", type=("build", "run"))
 
+    def patch(self):
+        filter_file(r"^\s*CC\s*=.*", "CC = " + self.compiler.cc, "configure/CONFIG.gnuCommon")
+        filter_file(r"^\s*CCC\s*=.*", "CCC = " + self.compiler.cxx, "configure/CONFIG.gnuCommon")
+
```

Comment:
This doesn't seem to be relevant to this PR's title. Could it be packaged separately?